### PR TITLE
Fix gosec target on prow

### PR DESCRIPTION
**Pull request by xpivarc**
**What this PR does / why we need it**:
Copy junit generated by gosec if junit file exists and we are on prow
Previous solution failed because we added GOSEC variable for running subset of tests

**Special notes for your reviewer**:

```release-note
None
```
/cc @dhiller

### CODE REVIEW

**Comment by dhiller** [comment]
gosec not finished yet: https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5179/pull-kubevirt-gosec/1369246199399321600

**Comment by xpivarc** [comment]
@dhiller working

**Review by dhiller — APPROVED** [review]
/approve

**Comment by qinqon** [comment]
"""
/hold
until emergency CI fixes of #5207 are merged
"""

**Comment by EdDev** [comment]
> """
> /hold
> until emergency CI fixes of #5207 are merged
> """

The mentioned PR is merged.

/hold cancel

**Pull request merged**
